### PR TITLE
Update best-practices.md

### PR DESCRIPTION
**Pull request by maikksmt**
Improved translation and corrected minor spelling errors.

- [x] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/gh-pages/CONTRIBUTING.md)?
- [x] Have you explained what your changes do, and why they add value to the Guides?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----

### CODE REVIEW

**Comment by Techassi** [comment]
Good work, gonna review it tomorrow. Check the review i made earlier. There are some things need to be changed.

File: _articles/de-DE/best-practices.md [hunk 1]
```diff
@@ -16,33 +16,33 @@ image: /assets/images/cards/best-practices.png
...
+Dokumentation beschreibt nicht nur deine Gedanken, sondern es hilft auch anderen Leuten zu verstehen was du erwartest oder brauchst, bevor sie überhaupt fragen.
 
 Dinge aufzuschreiben macht es einfacher „Nein“ zu sagen wenn etwas nicht in deinen Bereich reinpasst. Es macht es außerdem leichter für Leute einzusteigen und zu helfen. Du weißt nie wer dein Projekt liest oder benutzt.
 
 Wenn du keine Texte schreiben möchtest, sind Stichwörter ebenfalls besser als überhaupt nichts aufzuschreiben.
 
 ### Schreibe die Vision des Projekts auf
 
-Fange an mit dem Aufschreiben von den Zielen deines Projektes. Füge sie zu deiner README hinzu, oder erstelle eine VISION Datei. Wenn andere Artifakte helfen könnte, wie z.B. eine Roadmap, mache sie ebenfalls öffentlich.
+Fange an mit dem Aufschreiben von den Zielen deines Projektes. Füge sie zu deiner README hinzu, oder erstelle eine VISION Datei. Wenn andere Werkzeuge helfen können, wie z.B. ein Entwicklungsplan, mache sie ebenfalls öffentlich.
```
Comment (jens1o) [inline _articles/de-DE/best-practices.md]: Could you append a ` (Roadmap)` to `Entwicklungsplan`, please?

File: _articles/de-DE/best-practices.md [hunk 2]
```diff
@@ -157,31 +157,31 @@ Manchmal, wenn du „Nein“ sagst, wird dein potenzieller Beiträger vielleicht
...
   </p>
 </aside>
 
-Wenn du von deinem Projekt Abstand nehmen musst, entweder temporär oder permanent, gibt es keinen Grund sich zu schämen, wenn man fragt ob jemand anderes es übernehmen könnte.
+Wenn du von deinem Projekt Abstand nehmen musst, entweder temporär oder permanent, gibt es keinen Grund sich zu schämen wenn man fragt, ob jemand anderes es übernehmen könnte.
 
-Sofern andere Leute begeistert über die neue Richtung sind, gebe ihnen den Commit-Zugang oder reiche formal die Kontrolle weiter. Wenn jemand dein Projekt geforkt hat und es aktiv extern entwickelt, möchtest du das Projekt vielleicht in deinem Original Projekt verlinken. Es ist großartig dass soviele Leute dein Projekt am Leben haben möchten!
+Sofern andere Leute begeistert über die neue Richtung sind, gebe ihnen den Commit-Zugang oder reiche formal die Kontrolle weiter. Wenn jemand dein Projekt gegabelt hat und es aktiv extern entwickelt, möchtest du das Projekt vielleicht in deinem Original Projekt verlinken. Es ist großartig dass soviele Leute dein Projekt am Leben haben möchten!
```
Comment (jens1o) [inline _articles/de-DE/best-practices.md]: gegabelt sounds soo strange... :c

File: _articles/de-DE/best-practices.md [hunk 3]
```diff
@@ -197,15 +197,15 @@ Die Mitglieder deiner Community zu ermutigen auf ihrem Fork zu arbeiten kann das
 
 <aside markdown="1" class="pquote">
   <img src="https://avatars1.githubusercontent.com/u/481677?v=3&s=400" class="pquote-avatar" alt="avatar">
-  Ich pflege die 80:20 Regel. Wenn du einer von den Speziellen bist, forke bitte meine Arbeit. Ich werde nicht wütend! Meine öffentlichen Projekte sind immer dafür gedacht die wichtigsten Probleme zu lösen; Ich versuche es einfach zu machen um weitere Bereiche abzudecken mit dem Forken meiner Arbeit oder es schlichtweg zu erweitern.
+  Ich pflege die 80:20 Regel. Wenn du einer von den speziellen bist, dann gabel bitte meine Arbeit. Ich werde nicht wütend! Meine öffentlichen Projekte sind immer dafür gedacht die wichtigsten Probleme zu lösen; Ich versuche es einfach zu machen um weitere Bereiche abzudecken mit dem Forken meiner Arbeit oder es schlichtweg zu erweitern.
```
Comment (jens1o) [inline _articles/de-DE/best-practices.md]: again

**Comment by jens1o** [comment]
Thanks :)

**Pull request merged**